docs/root/operations/cli.rst

```diff
@@ -235,3 +235,9 @@ following are the command line options that Envoy supports.
 
   *(optional)* This flag disables Envoy hot restart for builds that have it enabled. By default, hot
   restart is enabled.
+
+.. option:: --filter-stats <string>
+
+  *(optional)* This flag prevents the creation of stats matching a regex. Since this flag is not set
+  by default, all stats are enabled by default. To disable all stats within Envoy,
+  call `--filter-stats ".*"`.
```

Comment:
IMO an allow/disallow hierarchy is a little easier to grok.

On another note, I assume we may also want this to be cluster-specific?

Reply:
Agreed, an allow/disallow hierarchy would be very clear. What about this?

```
./envoy                                         // all
./envoy --disallow-stats <regex>                // exclude
./envoy --disable-stats                         // none
./envoy --disable-stats --allow-stats <regex>   // include
```